### DISCUSSION TITLE
pdf download is only partially available

### DISCUSSION THREAD

**Discussion by AnniHuch** [knowledgefutures/pubpub#2901] [Support] [answered]
We are using pubpub as a way to publish our theoretical computer science lecture notes. Unfortunately, the download pdf feature only works partially. After a new release the pdf download is often not available for some amount of time and one day later it is magically working again. For me it seems like there is thrown some timeout or similar.
Is there any way to guarantee an available download at every (or most of) the time?
Thanks for some help in advance.

**Comment by gabestein** [comment, marked as answer]
Hi Anni — we recently fixed a small bug that was preventing some PDFs from generating right away. It can take a few minutes for them to generate after new releases.
